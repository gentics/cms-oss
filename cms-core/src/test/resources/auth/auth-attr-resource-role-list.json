{
  "roles": {
    "resource": {
      "cms" : [
        "cms-role-2"
      ]
    }
  }
}