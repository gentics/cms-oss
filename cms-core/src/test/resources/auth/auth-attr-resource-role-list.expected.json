{
  "4711": null,
  "Group2": ["1337"]
}